crates/topos/src/components/node/commands/up.rs

```diff
@@ -0,0 +1,14 @@
+use std::path::PathBuf;
+
+use clap::Args;
+use serde::Serialize;
+
+#[derive(Args, Debug, Serialize)]
+#[command(about = "Spawn your node!")]
+pub struct Up {
+    /// Name to identify your node
+    #[arg(long, env = "TOPOS_NODE_NAME", default_value = "default")]
+    pub name: Option<String>,
```

Comment:
If the `name` is present in every `node` command maybe it should be on the `node` root command. but we could update that later

Reply:
Will leave it for next PR!